src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -127,9 +131,11 @@ public Task Initialize()
                         var tw = new StringWriter();
                         msbuildProject.Save(tw);
                     }
-                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException)
+                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException ex)
                     {
                         // Indicate we weren't able to complete the action. We want to do a normal reload
+                        var faultEvent = new FaultEvent("vs.projectsystem.projectreload.reload-failed", "Project Reload Failure due to invalid project file", ex);
```

Comment:
by the way, VS telemetry API supports testing.

https://github.com/dotnet/roslyn-internal/pull/1432/files#diff-a366f8be34dd2bdad2da29250ce01fb5R997

you probably want to add a test to make sure telemetry is working at some point anyway.